scripts/Tools/case.build

```diff
@@ -99,7 +98,7 @@ def _main_func(description):
     elif(testname is not None):
         logging.warn("Building test for %s in directory %s" % (testname, caseroot))
         try:
-            test = globals()[testname](caseroot, case)
+            test = globals()[testname](caseroot=caseroot, case=case)
```

Comment:
It's a little dangerous to send keyword arguments via positional semantics, so I changed all these to keyword.